dev/MRTCore/mrt/mrm/mrmmin/mrmmin.vcxproj

```diff
@@ -376,14 +376,19 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\WindowsAppSDKInsightsResource\WindowsAppSDKInsightsResourceLib\WindowsAppSDKInsightsResourceLib.vcxproj">
```

Comment:
Reference should be to Insights lib and not InsightsResource lib

Reply:
The change that adds Microsoft.WindowsAppSDK.Insights.lib is still in review. As things are now, Chandra says this is correct.

Reply:
Done